api/app/models/spree/webhooks/subscriber.rb

```diff
@@ -1,6 +1,14 @@
 module Spree
   module Webhooks
     class Subscriber < Spree::Webhooks::Base
+      SUPPORTED_CUSTOM_EVENTS = {
```

Comment:
This could instead be added to every model decorator in which we're adding the logic for firing each event, no? This way you don't need. You could then just append right there the create/delete/update events to every model. What do you think @aleksandarpetrushev ?

Reply:
Do you mean for example in Webhooks::OrderDecorator, append to this list 'order.canceled', 'order.resumed' etc? And what do you mean append create/delete/update right there?

Reply:
Yes, for instance the Spree::Webhooks::OrderDecorator can have it's array of supported custom events (`%w[order.canceled order.paid order.placed order.resumed order.shipped]`) hardcoded and then you can just append what you do in `default_events` - the common create/delete/update events. This way it makes every decorator to have knowledge of its own events and allows the Subscriber to "know less" about what other models do.

You could even define the `default_events` in the HasWebhooks concern and just get rid of the `events_list_for` method, I think. Then the `supported_events` would be all the events declared in every model - whether custom or the common ones and the Subscriber model can just map them.

Reply:
yes, that sounds good